credsweeper/utils/util.py

```diff
@@ -59,3 +61,84 @@ def get_shannon_entropy(cls, data: str, iterator: Chars) -> float:
                 entropy += -p_x * math.log(p_x, 2)
 
         return entropy
+
+    @classmethod
+    def read_patch_file(cls, path: str) -> List[str]:
+        with open(path, "r") as patch_file:
+            diff_data = patch_file.readlines()
+        return diff_data
+
+    @classmethod
+    def patch2files_diff(cls, raw_patch: List[str], change_type: str) -> Dict[str, List[Dict]]:
+        """
+        Generates files rows from diff with only added and deleted lines (e.g. marked + or - in diff)
```

Comment:
```suggestion
        """Generates files rows from diff with only added and deleted lines (e.g. marked + or - in diff)
```
All docstring should start on the sample line as `"""` due to https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
"The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. "

We do not follow this convention in all functions, but it would be nice to start...